pkg/doozer/prepare/prepare.go

```diff
@@ -0,0 +1,160 @@
+package prepare
```

Comment:
Det samme gjelder denne filen. Det begynner å bli en vis grad av duplisering i kodebasen

Reply:
Opprettet en sak for refaktorering av duplisert kode (AOT-718)